d3-geo/index.d.ts

```diff
@@ -936,6 +936,16 @@ export interface GeoPath<This, DatumObject extends GeoPermissibleObjects> {
     centroid(object: DatumObject): [number, number];
 
     /**
+     * Returns the projected planar length (typically in pixels) for the specified GeoJSON object.
+     * Point and MultiPoint features have zero length. For Polygon and MultiPolygon features, this method computes the summed length of all rings.
+     *
+     * This method observes any clipping performed by the projection.
+     *
+     * @param object GeoJSON object to measure.
```

Comment:
Minor detail: For the other methods on `GeoPath` we started the `@param` description with _An object..._, We should keep it consistent, as the `DatumObject` does not have to be equivalent to the narrow definition of `GeoJSON object`.